shop/src/app/components/presentational/products-details-view/products-details-view.component.scss

```diff
@@ -0,0 +1,21 @@
+.top {
```

Comment:
Related to what I have mentioned earlier in the HTML file, a proper naming would be more in handy

Reply:
Done